.github/workflows/notify_test_workflow.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     steps:
       - name: "Notify test workflow"
         uses: actions/github-script@v3
-        if: ${{ github.base_ref == 'master' }}
+        if: github.base_ref == 'master' && github.repository == 'apache/spark'
```

Comment:
@Yikun so is the problem that the "Notify test workflow" is triggered in the forked repository's PRs too? I think it's not a big problem but okay.

Reply:
oops, I am wrong to thought that the fork repo "Notify test workflow" will create a new series workflow, but actually, the notify test workflow just found the job and give the link.

So, as you said, this pr only skip the notify process.